README.md

```diff
@@ -15,6 +15,8 @@ See build(7), config(8), [FreeBSD handbook on building userland](https://docs.fr
 For information on the CPU architectures and platforms supported by FreeBSD, see the [FreeBSD
 website's Platforms page](https://www.freebsd.org/platforms/).
 
+Want to snag an ISO of FreeBSD? Head to the officials [releases page](https://download.freebsd.org/ftp/releases/ISO-IMAGES/) to get yourself one. 
+
```

Comment:
Love the information. 
I'd suggest a little more formal phrasing:

```
Official FreeBSD bootable images on our [releases page](...).
```

Reply:
Iso is becoming a bit of an anachronism

Reply:
Made the change. How does it look?